src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -461,6 +461,16 @@ public static IEnumerable<object[]> Cookies_TestData()
                 }
             };
 
+            // Not added if the cookie already exists.
+            yield return new object[]
+            {
+                "cookie: name=value,name=value", new CookieCollection
+                {
+                    new Cookie("name", "value")
+                }
+            };
+
+            /*[ActiveIssue(18486)]
```

Comment:
This looks like a pretty bad issue with CookieParser in .NET Core: it doesn't seem to support strings like "name1=value1,name2=value2;name3=value3"
This is tracked by #18486